src/decl/decl-ok.rkt

```diff
@@ -133,6 +133,31 @@
    (where/error (WhereClause ...) WhereClauses)
    ]
 
+  [;; For a static declared in the crate C, like the following:
+   ;;
+   ;;     static NAMED<T>: Foo<T> where T: Trait;
+   ;;
+   ;; we require that the type is well formed assuming the where clauses are satisfied
+   ;; and that the type is `Send`.
+   (crate-item-ok-goal CrateDecls (StaticId (static KindedVarIds WhereClauses Ty)))
+   (∀ KindedVarIds
+      (implies
+       (; assuming all generic parameters are WF...
+        (well-formed KindedVarId) ...
+        ; ...where-clauses are satisfied...
+        (where-clause->hypothesis WhereClause) ...)
+       (&& ((; ... then the trait must be implemented
+             well-formed (type Ty))
+            (; ... and the type must be Send
+             is-implemented (rust:Send (Ty)))
```

Comment:
Statics don't have to be Send, but they do have to be Sync:
```rust
struct Foo(*const ());
// unsafe impl Send for Foo {}
unsafe impl Sync for Foo {}
static FOO: Foo = Foo(0 as _); // OK
```

Reply:
Whoops :)